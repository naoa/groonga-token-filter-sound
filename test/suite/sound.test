plugin_register token_filters/sound

tokenize TokenDelimit \
  "あい 々あいうえ 今日" \
  --token_filters TokenFilterSound
