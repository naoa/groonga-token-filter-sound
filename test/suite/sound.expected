plugin_register token_filters/sound
[[0,0.0,0.0],true]
tokenize TokenDelimit   "あい 々あいうえ 今日"   --token_filters TokenFilterSound
[[0,0.0,0.0],[{"value":"今日","position":2,"force_prefix":false}]]
